src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/prisons/SupportedPrisonsRepository.kt

```diff
@@ -34,7 +44,7 @@ data class SupportedPrison(
   override fun hashCode(): Int = javaClass.hashCode()
 
   @Override
-  override fun toString(): String {
-    return this::class.simpleName + "(code = $code, active = $active, updatedBy = $updatedBy, updated = ${DateTimeFormatter.ISO_DATE_TIME.format(updated)} )"
-  }
+  override fun toString(): String =
+    this::class.simpleName + "(code = $code, active = $active, updatedBy = ${updatedBy ?: "null"}, " +
+      "updated = ${updated?.let { DateTimeFormatter.ISO_DATE_TIME.format(updated) } ?: "null"} )"
```

Comment:
Should never be null in real life but might be if somebody has broken a test or introduced a bug